content/docs/command-reference/run.md

```diff
@@ -151,7 +151,7 @@ variables in it that should be evaluated dynamically. Examples:
 
 ```dvc
 $ dvc run -n my_stage "./my_script.sh > /dev/null 2>&1"
-$ dvc run -n my_stage './my_script.sh $MYENVVAR'
+$ dvc run -n my_stage -f './my_script.sh $MYENVVAR'
```

Comment:
let's use `--force` (other commands have `-f` for `file`, it's better to be explicit)

Reply:
also, I thinks it's better to use a different name for the stage in this case

Reply:
There's no `-f` short flag for `--file` (`add` or `import*`). I think because I nagged the core team about flag naming consistency at some point last year 😬 

But sure, it makes the example self-documenting. Will do ⌛

Reply:
Oh nvmd I see you suggested to use a different name. Agree. OK, done!

Reply:
PTAL